cli/azd/internal/repository/detect_confirm.go

```diff
@@ -317,11 +350,6 @@ func (d *detectConfirm) add(ctx context.Context) error {
 				fmt.Sprintf("%s in %s", projectDisplayName(svc), filepath.Base(svc.Path)))
 		}
 
-		svcSelect, err = tabWrite(svcSelect, 3)
```

Comment:
Removing this as it was effectively dead code since there were no tab characters in the selection. This changed awhile back.